packages/react/src/utils/useOutboundClickListener.tsx

```diff
@@ -10,7 +10,7 @@ const useOutboundClickListener = ({ trackLink }: MatomoInstance): void => {
     const { target } = event
     const { nodeName } = target as HTMLElement
 
-    if (nodeName === 'A' && trackLink) {
+    if (nodeName === 'A' && matomoInstance.trackLink) {
```

Comment:
Do we need to check for the presence of `matomoInstance.trackLink` here? It should always be present.